sdk/framework/backend.go

```diff
@@ -527,10 +528,12 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 	// the request, the type will be used as part of the request/response body
 	// names in the OAS document.
 	requestResponsePrefix := req.GetString("requestResponsePrefix")
+	dynamicPaths := fmt.Sprintf("%v", req.Data["dynamicPaths"])
+	dynamic, _ := strconv.ParseBool(dynamicPaths)
 
 	// Build OpenAPI response for the entire backend
 	doc := NewOASDocument()
-	if err := documentPaths(b, requestResponsePrefix, doc); err != nil {
+	if err := documentPaths(b, requestResponsePrefix, dynamic, doc); err != nil {
```

Comment:
I feel like `custom` is a little confusing. B/c you mount engines at a `custom` path, so it almost sounds like the opposite to me. Like it would use the hardcoded ones in this case.

we could do `dynamicMountPaths` and update everything accordingly?